web/resources/user/admin/views/AdminCreateUser.js.jsx

```diff
@@ -61,7 +61,7 @@ class AdminCreateUser extends Binder {
 
   render() {
     const { user } = this.state;
-    const isEmpty = !user || (user.username === null || user.username === undefined);;
+    const isEmpty = !user || (user.username === null || user.username === undefined);
```

Comment:
I think this always resolves as `isEmpty === true`.  maybe make it just `!user`

Reply:
I thought the same thing, but we actually define the default user in the reducer with empty strings for those values, so it does work. But it is cleaner to make it just `!user`